supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -590,6 +590,28 @@ Use "Red Hat Virtualization Manager". Spell out in full for the first instance i
 
 *See also*: xref:zone-group[zone group]
 
+[discrete]
+[[regex]]
+==== image:images/no.png[no] regex (noun)
+*Description*: _Regex_ is an abbreviation for "regular expression". Do not use "regex" as a replacement for "regular expression".
+
+*Use it*: no
+
+*Incorrect forms*:
+
+*See also*: xref:regular-expression[regular expression]
+
+[discrete]
+[[regular-expression]]
+==== image:images/yes.png[yes] regular expression (noun)
+*Description*: A _regular expression_ is a string of text specifying a pattern that a computer can use when searching a body of text to match characters or groups of characters. 
```

Comment:
```suggestion
*Description*: A _regular expression_ is a string of letters, numbers, and symbols that defines a pattern. When searching a body of text, software can use this pattern to match characters or groups of characters and return results.
```
- To my ears, this sentence seems a bit too long. Maybe consider shortening it by breaking it into two sentences. My edit shows one possibility.
- Consider saying `string of letters, numbers, and symbols` rather than `text` to define the term more specifically. Your call...
- Consider moving the `when searching` clause in front of `a computer` to simplify the syntax and to avoid possibly unclear modifiers in the dependent clauses.
- Consider specifying `computer program` or `software` rather than just `computer` because computer hardware by itself can't really perform searches without software.
- Consider adding `and return results` to the end because software not only matches the defined pattern but returns the results for further processing.